query/src/main/scala/filodb/query/LogicalPlan.scala

```diff
@@ -144,28 +152,59 @@ case class ScalarVectorBinaryOperation(operator: BinaryOperator,
   */
 case class ApplyInstantFunction(vectors: PeriodicSeriesPlan,
                                 function: InstantFunctionId,
-                                functionArgs: Seq[Any] = Nil) extends PeriodicSeriesPlan with NonLeafLogicalPlan {
+                                functionArgs: Seq[FunctionArgsPlan] = Nil) extends PeriodicSeriesPlan
+  with NonLeafLogicalPlan {
   override def children: Seq[LogicalPlan] = Seq(vectors)
 }
 
 /**
   * Apply Miscellaneous Function to a collection of RangeVectors
   */
 case class ApplyMiscellaneousFunction(vectors: PeriodicSeriesPlan,
-                                function: MiscellaneousFunctionId,
-                                functionArgs: Seq[Any] = Nil) extends PeriodicSeriesPlan with NonLeafLogicalPlan {
+                                      function: MiscellaneousFunctionId,
+                                      stringArgs: Seq[String] = Nil) extends PeriodicSeriesPlan
+  with NonLeafLogicalPlan {
   override def children: Seq[LogicalPlan] = Seq(vectors)
 }
 
 /**
   * Apply Sort Function to a collection of RangeVectors
   */
 case class ApplySortFunction(vectors: PeriodicSeriesPlan,
-                                      function: SortFunctionId,
-                                      functionArgs: Seq[Any] = Nil) extends PeriodicSeriesPlan with NonLeafLogicalPlan {
+                             function: SortFunctionId) extends PeriodicSeriesPlan with NonLeafLogicalPlan {
   override def children: Seq[LogicalPlan] = Seq(vectors)
 }
 
+//scalastyle:off
```

Comment:
Can we please not turn off scalastyle here?  Thx.... don't see any reason why we should need to

Reply:
Without it, I get error ```Number of types declared in the file exceeds 30```
Adding a comment for this

Reply:
ok,  can we add a specific disabling just the number of types?  Thx